MaxiOps/vlaura_voting/2024-05-24-vote-report.txt

```diff
@@ -0,0 +1,21 @@
+Voting for: {
```

Comment:
would be nice if you could use Pandas or something to put this into a table or csv form.

Would also be amazing to add the number of vlAURA votes that equated to, but not critical.